packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -96,6 +82,16 @@ type TProjectPermissions = {
 </DataMenu>
 */
 
+const HeartIcon = lazy(() => import('./legacy-icons/heart'));
+const PaperclipIcon = lazy(() => import('./legacy-icons/paperclip'));
+const PluginIcon = lazy(() => import('./legacy-icons/plugin'));
+const RocketIcon = lazy(() => import('./legacy-icons/rocket'));
+const StarIcon = lazy(() => import('./legacy-icons/star'));
+const ConnectedSquareIcon = lazy(
+  () => import('./legacy-icons/connected-square')
+);
+const WorldIcon = lazy(() => import('./legacy-icons/world'));
```

Comment:
Unrelated changes to the redesign but I thought to code split the legacy icons to not include them in the bundle by default.